src/server.tsx

```diff
@@ -181,9 +181,8 @@ app.post(
             return;
         }
 
-        const shouldNotRender = shouldNotRenderEpic(targeting);
         const tests = await fetchConfiguredEpicTestsCached();
-        const got = shouldNotRender ? undefined : findVariant(tests, targeting);
+        const got = findVariant(tests, targeting);
```

Comment:
One of the variant filters is `shouldNotRenderEpic` - this check doesn't need to happen in both places